src/catalog/src/durable/upgrade/v53_to_v54.rs

```diff
@@ -0,0 +1,441 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use mz_stash::upgrade::WireCompatible;
+use mz_stash::wire_compatible;
+
+use crate::durable::upgrade::MigrationAction;
+use crate::durable::upgrade::{objects_v52 as v52, objects_v54 as v54};
```

Comment:
shouldn't these be v53, not v52?

Reply:
Eeek, yes, good catch. Looks like I missed this when resolving conflicts.

Reply:
Fixed